src/components/expanded-state/UniqueTokenExpandedState.tsx

```diff
@@ -174,13 +173,11 @@ const Markdown = ({
 interface UniqueTokenExpandedStateProps {
   asset: UniqueAsset;
   external: boolean;
-  lowResUrl: string;
 }
 
 const UniqueTokenExpandedState = ({
   asset,
   external,
-  lowResUrl,
```

Comment:
all lowResUrl handling is now down in `UniqueTokenImage` instead of outside of it